test/integration/marked-spec.js

```diff
@@ -1,18 +1,51 @@
-var marked = require('../../marked.min.js');
+var sut = require('../../marked.min.js');
```

Comment:
why `sut`?

Reply:
System Under Test. 

The hope is to distinguish it from the other modules which are there to aid in testing while not being "under test".

Reply:
See also: https://en.wikipedia.org/wiki/System_under_test